handlers/templates/pages/guest-link-index.html

```diff
@@ -29,7 +29,11 @@
 
 {{ define "script-tags" }}
```

Comment:
Can we add an e2e test to cover this new functionality to `e2e/guest-links.spec.ts`?